Containers/nextcloud/cron.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 set -eu
 
-while true; do
-    php -f /var/www/html/cron.php &
-    sleep 5m
-done
+trap 'true' SIGINT SIGTERM
+
+exec /cronjob.sh &
+
+wait $!
+
+echo "Cronjob successfully exited."
```

Comment:
what does wait $! does diffrent to just wait?

Reply:
Honestly not sure

Reply:
and where do you got this?

Reply:
From here: https://github.com/nextcloud/all-in-one/blob/722a96828e0a5b69cb9fcda5c14b75df3035a0c8/Containers/postgresql/start.sh#L169

Reply:
and where does this come from?

Reply:
I think somehwere from stackoverflow or so...

Reply:
but did not note that down...

Reply:
from what I know is wait either waits for a specified pid to exit or "all" (I think all launched by the shell script)

Reply:
okay, but that would be what we want here or not?

Reply:
> I think somehwere from stackoverflow or so...

https://stackoverflow.com/questions/33375924/what-does-wait-mean

The answer was "($!) Expands to the process ID of the job most recently placed into the background, whether executed as an asynchronous command or using the bg builtin"

Reply:
I don't know if this will help but...

https://superuser.com/questions/959508/cron-job-does-not-wait-end-of-process

They are suggesting to remove "wait " because it will be obsolete. Instead to use "sleep 2".

Reply:
I hope this will work in Docker Desktop for Windows. As far as I can tell, DDfW gets TASKKILL and says "Ok" instead of wait x number of seconds. Might have to send this up to Docker.

Reply:
This seems more appropriate: https://stackoverflow.com/questions/356100/how-to-wait-in-bash-for-several-subprocesses-to-finish-and-return-exit-code-0

Reply:
Will have a look, thanks!

Reply:
You could then put those wait calls in a cleanup function and have it called by the trap